datanode/sqlstore/migrations/0069_mark_price.sql

```diff
@@ -0,0 +1,130 @@
+-- +goose Up
+create type composite_price_type as enum('COMPOSITE_PRICE_TYPE_UNSPECIFIED','COMPOSITE_PRICE_TYPE_WEIGHTED','COMPOSITE_PRICE_TYPE_MEDIAN','COMPOSITE_PRICE_TYPE_LAST_TRADE');
+alter table market_data add column if not exists mark_price_type composite_price_type,
+                        add column if not exists index_price_type composite_price_type,
+                        add column if not exists index_price HUGEINT,
+                        add column if not exists next_index_price_calc timestamp with time zone;
+alter table current_market_data add column if not exists mark_price_type composite_price_type,
+                                add column if not exists index_price_type composite_price_type,
+                                add column if not exists index_price HUGEINT,
+                                add column if not exists next_index_price_calc timestamp with time zone;
+
+update market_data set mark_price_type = 'COMPOSITE_PRICE_TYPE_LAST_TRADE', 
+                       index_price_type = 'COMPOSITE_PRICE_TYPE_UNSPECIFIED',
+                       index_price ='0';
+
+update current_market_data set mark_price_type = 'COMPOSITE_PRICE_TYPE_LAST_TRADE',
+                               index_price_type = 'COMPOSITE_PRICE_TYPE_UNSPECIFIED',
+                               index_price ='0';
+
+
+-- +goose StatementBegin
+UPDATE proposals 
+SET terms = jsonb_set(
+  terms, 
+  '{terms, updateMarket, changes}', 
+  terms #> '{terms, updateMarket, changes}' || '{"markPriceConfiguration": {"decayWeight": "0", "decayPower": 0, "cashAmount":"0","compositePriceType":"COMPOSITE_PRICE_TYPE_UNSPECIFIED"}}'
```

Comment:
Rather than unspecified, shouldn't we set them to `COMPOSITE_PRICE_TYPE_LAST_TRADE` instead?

Reply:
done